systems/controllers/iiwa_position_controller.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include "common/find_resource.h"
+#include "systems/framework/timestamped_vector.h"
+#include "drake/systems/framework/leaf_system.h"
+#include "drake/systems/framework/basic_vector.h"
+#include "drake/multibody/parsers/urdf_parser.h"
+#include "drake/multibody/rigid_body_plant/rigid_body_plant.h"
+#include "drake/manipulation/util/sim_diagram_builder.h"
+
+#include <iostream>
+#include <fstream>
+
+using Eigen::VectorXd;
+using Eigen::MatrixXd;
+using Eigen::Quaternion;
+using Eigen::Quaterniond;
+using Eigen::AngleAxis;
+using Eigen::AngleAxisd;
+using drake::systems::LeafSystem;
+using drake::systems::Context;
+
+namespace dairlib{
+namespace systems{
+
+class KukaIiwaPositionController : public LeafSystem<double> {
```

Comment:
Class needs a description of what it does. PD control on end effector position and orientation, with orientation specified as a quaternion, etc.